pyglidein/startd_cron_scripts/clsim_gpu_test.py

```diff
@@ -47,7 +55,7 @@ def main():
         print 'PYGLIDEIN_RESOURCE_GPU=True'
         print '- update:true'
     except SystemExit:
```

Comment:
How does `SystemExit` get generated?  This is an unusual exception to catch.

Reply:
sys.exit() inside a try except statement will generate this.  This is going away so the exception handler will too.